### PR TITLE
Give some indications that a worker is modified

### PR DESCRIPTION
**Pull request by vdbergh**
This is the server counter part to #1442 .

Add (M) to workername of modified worker.
Add + to version of modified worker in machine table.

### CODE REVIEW

File: server/fishtest/templates/machines_table.mak [hunk 1]
```diff
@@ -20,7 +20,7 @@
           gcc_version = ".".join([str(m) for m in machine['gcc_version']])
           compiler = machine.get('compiler', 'g++')
           python_version = ".".join([str(m) for m in machine['python_version']])
-          version = machine['version']
+          version = str(machine['version'])+("+" if machine['modified'] else "")
```
Comment (ppigazzini) [inline server/fishtest/templates/machines_table.mak:23]: Black adds spaces around the plus sign:
```python
version = str(machine['version']) + ("+" if machine['modified'] else "")
```
Other options are still ugly for the flood of single/double quotes, parentheses/brackets/braces, the last one is funny :)
```python
version = "".join([str(machine['version']), "+" if machine['modified'] else ""])
version = f"{machine['version']}{'+' if machine['modified'] else ''}"
version = "{}{}".format(machine['version'], "+" if machine['modified'] else "")
version = str(machine['version']) + '+' * machine['modified']
```

Comment (vdbergh) [reply server/fishtest/templates/machines_table.mak:23]: The last one is short but it is a bit of a hack (it depends on the fact that Python interprets True as 1 and False a 0). But I guess it is clear what it is meant. I will take it.

Comment (vdbergh) [reply server/fishtest/templates/machines_table.mak]: Done (not tested since this laptop doesn't have fishtest).

**Comment by ppigazzini** [comment]
DEV updated, LGTM.
"183+" seems positive ("I run the plus version of the worker"), perhaps it's better to use another symbol: "183<", "183#", "183*"

File: server/fishtest/templates/machines_table.mak (hunk 1 above)
Comment (ppigazzini) [reply server/fishtest/templates/machines_table.mak]: DEV updated, it works.

**Comment by dav1312** [comment]
> DEV updated, LGTM. "183+" seems positive ("I run the plus version of the worker"), perhaps it's better to use another symbol: "183<", "183#", "183*"
> 
> EDIT: or letter, "183M" or "183m" to be consistent with the renaming of the worker.

I think the best symbol is `*`, the other ones seem confusing.

**Comment by vdbergh** [comment]
Maybe also for the worker name (instead om (M))?

**Comment by vdbergh** [comment]
I changed both of them to *. AFK for an hour or so.

**Review by ppigazzini — APPROVED** [review]
DEV updated, LGTM, logs clean.

**Comment by ppigazzini** [comment]
PROD updated, thank you @vdbergh :)

**Comment by ppigazzini** [comment]
Modders caught! :)
image

**Comment by vdbergh** [comment]
That's a bit weird though. I would have thought this would only happen for developers (why would one otherwise modify the worker). Also technologov seems to have some modded workers, but not all of them are modded.

**Comment by ppigazzini** [comment]
Asked to technologov and sebastronomy on Discord.

**Comment by ppigazzini** [comment]
BabyShark on Discord pointed out that the raw data on Github can have a 5 minutes delay.
https://stackoverflow.com/questions/62785962/get-raw-file-from-github-without-waiting-for-5-minute-cache-update
https://stackoverflow.com/questions/46551413/github-not-update-raw-after-commit
> GitHub serves "raw" pages with Cache-Control: max-age=300. That's specified in seconds, meaning the pages are intended to be cached for 5 minutes.
You can see this if you open the Developer Tools in your web browser of choice before clicking the "Raw" button on GitHub.

**Pull request merged**